app/docs/0.10.x/admin-api.md

```diff
@@ -1438,5 +1439,67 @@ HTTP 200 OK
 }
 ```
 
+---
+
+### List active targets
+
+Retrieve a list of active targets (targets whose most recent weight is not 0)
+under a given upstream.
+
+### Endpoint
+
+<div class="endpoint get">/upstreams/{name or id}/targets/active</div>
+
+Attributes | Description
+---:| ---
+`name or id`<br>**required** | The unique identifier **or** the name of the upstream for which to list the targets
+
+#### Response
+
+```
+HTTP 200 OK
+```
+
+```json
+{
+    "total": 2,
+    "data": [
+        {
+            "created_at": 1485524883980,
+            "id": "18c0ad90-f942-4098-88db-bbee3e43b27f",
+            "target": "127.0.0.1:20000",
+            "upstream_id": "07131005-ba30-4204-a29f-0927d53257b4",
+            "weight": 100
+        },
+        {
+            "created_at": 1485524914883,
+            "id": "6c6f34eb-e6c3-4c1f-ac58-4060e5bca890",
+            "target": "127.0.0.1:20002",
+            "upstream_id": "07131005-ba30-4204-a29f-0927d53257b4",
+            "weight": 200
+        }
+    ]
+}
+```
+
+---
+
+### Delete target
+
+#### Endpoint
+
+<div class="endpoint delete">/upstreams/{name or id}/targets/{target}</div>
+
+Attributes | Description
+---:| ---
+`name or id`<br>**required** | The unique identifier **or** the name of the upstream for which to delete the target
+`target`<br>**required** | The host/port combination element of the target to remove
```

Comment:
all of those need dots at the end for consistency (same of active targets attributes)